scrapers/WankzVR.yml

```diff
@@ -0,0 +1,25 @@
+name: "WankzVR"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - wankzvr.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="detail"]
+    scene:
+      Title: $info//h1/text()
+      Date:
+        selector: $info//span[@class="detail__date"]/text()
+        parseDate: 2 January, 2006
+      Details:
+        selector: //div[@class="detail__txt detail__txt-show_lg"]/text()|//span[@class="more__body"]/text()
+        concat: " "
+      Tags:
+        Name: $info//div[@class="tag-list__body"]//a/text()
+      Performers:
+        Name: $info//div[@class="detail__models"]/a/text()
```

Comment:
Performers name get matched twice also.
`Name: //div[@class="detail__inf detail__inf-align_right"]/div[@class="detail__models"]/a/text()` 
seems to work